core/src/lib.rs

```diff
@@ -44,5 +44,7 @@ pub mod reservation;
 pub mod str;
 pub mod tasks;
 
+pub use ids::{hash, hash_of_array};
```

Comment:
imo this better to be moved somewhere like utils mod/root rather than just reexported

Reply:
done